java/server/src/org/openqa/selenium/grid/sessionqueue/NewSessionQueue.java

```diff
@@ -66,6 +76,40 @@ public boolean hasRequestTimedOut(HttpRequest request) {
     return duration.compareTo(requestTimeout) > 0;
   }
 
+  protected Capabilities validateCaps(Capabilities caps) {
+    String browser = caps.getBrowserName().isEmpty() ? "ANY" : caps.getBrowserName();
+    Platform platform = caps.getPlatformName() == null ? Platform.ANY : caps.getPlatformName();
```

Comment:
Again, prefer to omit this option if it's empty

Reply:
https://github.com/SeleniumHQ/selenium/pull/9043/#discussion_r560822950